src/styles/blocks/product-card.css

```diff
@@ -0,0 +1,89 @@
+.product-card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: 200px;
```

Comment:
<img width="1016" alt="image" src="https://user-images.githubusercontent.com/102411130/202842759-ec29d324-a67c-4d58-86e6-1c8aeb74c44a.png">

Reply:
check the checklist

Reply:
also don't forget about css properties order
http://codeguide.academy/html-css.html#css-order